libs/balances-and-allowances/src/utils/getIsBlockNumberRelevant.ts

```diff
@@ -0,0 +1,9 @@
+export function getIsBlockNumberRelevant<T = number | undefined>({
+  blockNumber,
+  prevBlockNumber,
+}: {
+  blockNumber: T
+  prevBlockNumber: T
+}): boolean {
+  return typeof prevBlockNumber === 'number' && typeof blockNumber === 'number' ? blockNumber > prevBlockNumber : true
```

Comment:
Here, if `prevBlockNumber` is falsy, it'll return false.
Is that what we want if `blockNumber` is a valid one?